paasta_tools/contrib/get_running_task_allocation.py

```diff
@@ -275,54 +275,57 @@ def parse_args() -> argparse.Namespace:
         "--scheduler",
         help="Scheduler to get task info from",
         dest="scheduler",
-        default="mesos",
+        default="kubernetes",
         choices=["mesos", "kubernetes"],
     )
+    parser.add_argument(
+        "--additional-namespaces-exclude",
+        help="full names of namespaces to not fetch allocation info for those that don't match --namespace-prefix-exlude",
+        dest="additional_namespaces_exclude",
+        nargs="+",
+        default=[],
+    )
     parser.add_argument(
         "--namespace-prefix",
-        help="prefix of the namespace to fetch the logs for"
+        help="[UNUSED] prefix of the namespace to fetch the logs for"
```

Comment:
i'll remove this arg after this is running in prod

Reply:
```suggestion
        help=argparse.SUPPRESS,
```
to make the help go away